docs/pages/api-reference/liveblocks-react.mdx

```diff
@@ -1130,6 +1130,9 @@ import { useThreads } from "./liveblocks.config";
 const { threads, error, isLoading } = useThreads();
 ```
 
+👉 A [Suspense version][] of this hook is also available, which will never
```

Comment:
I need to remove all these emoji! But yes, this is consistent with the rest of the page, so we'll leave it as it is for now, thanks 🙂